src/workflows-app/components/WorkflowCard.ts

```diff
@@ -48,6 +51,10 @@ export type WorkflowCardProps = {
 
 export const WorkflowCard = ({ method, subCard, children }: PropsWithChildren<WorkflowCardProps>) => {
   const isWorkflowSet = 'methods' in method; // Used to narrow type
+  const isPrivate = isWorkflowSet ? false : method.isPrivate;
```

Comment:
I think `method.isPrivate` could be null if it's not a GitHub method - am I right about that?

If so, the downstream code might work even if it can be null (I'm not sure), but it's probably better to make sure it gets turned to `false`

Reply:
Ah, good catch. I'll add a test for that as well.